src/compose/ComposeBox.js

```diff
@@ -432,49 +502,30 @@ class ComposeBoxInner extends PureComponent<Props, State> {
     dispatch(sendTypingStop(destinationNarrow));
   };
 
-  inputMarginPadding = {
-    paddingHorizontal: 8,
-    paddingVertical: Platform.select({
-      ios: 8,
-      android: 2,
-    }),
-  };
+  getValidationErrors = (): $ReadOnlyArray<ValidationError> => {
+    const { mandatoryTopics } = this.props;
+    const destinationNarrow = this.getDestinationNarrow();
+    const { message } = this.state;
+
+    const result = [];
 
-  styles = {
-    wrapper: {
```

Comment:
ComposeBox [nfc]: Bring out `componentStyles`

I'd rather keep the styles here where they are -- it means they're there right next to the code in the `render` method that refers to them. Otherwise they end up separated from that code by a few hundred lines of interaction logic, which doesn't really care about the layout details or vice versa.